dockers/docker-snmp-sv2/snmpd.conf.j2

```diff
@@ -14,6 +13,7 @@
 #  AGENT BEHAVIOUR
 #
 
+{% if minigraph_mgmt_interface.addr | ipv4 %}
```

Comment:
You can't bind on any other addresses with port 161 anymore after you bind on 0.0.0.0:161

---
In reply to: [99453733](https://github.com/Azure/sonic-buildimage/pull/253#discussion_r99453733) [](ancestors = 99453733)